### DISCUSSION TITLE
Why can't we use it in production?

### DISCUSSION THREAD

**Discussion by dcosmin2003** [Adyen/adyen-react-native#50] [General]
I'm just curious if it's because is not tested enough, or something else?

**Comment by descorp** [comment]
Hey @dcosmin2003 

Thanks for asking :)

We always want to be transparent with our merchants (you).
Firs of all - we don't want to release a product to a large audience that is not yet matching our quality standarts. Before releasing it, we want to make sure that there is proper documentation and that all the key functionalities work as intended.
Second - we believe that until we have a confirmed case of our SDK live and kicking for a real merchant, we can't truly say it is "market ready."

BTW kudos to those brave merchants that helping us pioneer this SDK! You are the real MVP 💚

**Reply by chrispaynter** [reply]
Hi @descorp,

We'll be using Adyen on 3 high profile projects in the new year, multiple 1000s of payment transactions per hour per project. I dare say it'd be exactly the type of use cases you'd be looking for to be able to say that this SDK is market ready.

We've integrated Adyen before in React Native for other projects, but rolled our own RN SDK due to there being no official one a few years ago. 

We'd be interested in using this official SDK moving forward, though it would be really helpful if we could put an architect on your side and our side together so that we can understand a bit more about it's current state and how we could potentially help you move this SDK forward.

Can we arrange this?

**Reply by descorp** [reply]
Hey @chrispaynter 

Absolutely! We are always welcome to feedback 💚

You can reach out to us via your Account Manager, support form in your Customer Area, or [Adyen Support](https://www.adyen.help/hc/en-us/requests/new?ticket_form_id=4404760485788).
We don't (yet) have a dedicated "React Native" integration or topic, so you can use "iOS" or "Android" for now.
Use the subject "Feedback to Checkout Crossplatform team" and add a link to this thread :)
